tdcommon/src/main/java/thredds/server/catalog/tracker/DatasetTrackerChronicle.java

```diff
@@ -132,6 +132,9 @@ private void open() throws IOException {
     ChronicleMapBuilder<String, DatasetExt> builder = ChronicleMapBuilder.of(String.class, DatasetExt.class)
         .averageValueSize(averageValueSize).entries(maxDatasets).averageKeySize(averagePathLength)
         .valueMarshaller(DatasetExtBytesMarshaller.INSTANCE).skipCloseOnExitHook(true);
+    if (builder == null) {
+      throw new IOException("Failed to create Dataset catalog tracker");
+    }
```

Comment:
This check might be superfluous. I'm not sure if ChronicleMapBuilder.of() can even return a null. At the moment for my situation, it throws an exception instead.